src/ir_cache/IRModuleCache.cpp

```diff
@@ -118,7 +118,7 @@ Runtime::ModuleRef IRModuleCache::getCompiledMainModule(const std::string& user,
             }
         }
     } else {
-        logger->debug("Using cached compiled main module {}", key);
+        logger->debug("Using cached compiled main module {}/{}", user, func);
```

Comment:
The logging in this file was hard to read as it used `key` rather than the `user/func` format